vhdbuilder/packer/install-dependencies.sh

```diff
@@ -37,7 +37,7 @@ installDeps
 cat << EOF >> ${VHD_LOGS_FILEPATH}
   - apache2-utils
   - apt-transport-https
-  - blobfuse=1.3.5
+  - blobfuse=1.3.7
```

Comment:
This is just log, are you missing the real change?

Reply:
fixed